auth0/src/main/java/com/auth0/android/provider/CustomTabsController.java

```diff
@@ -119,7 +119,7 @@ public void launchUri(@NonNull final Uri uri, final boolean launchAsTwa, ThreadS
             return;
         }
 
-        new Thread(() -> {
+        threadSwitcher.backgroundThread(() -> {
```

Comment:
As the threadSwitcher param can be null, do we need a check here? And what do we do if it is null?

Reply:
Hey @jimmyjames `CustomTabsController` is not public and used only internally only from one place. So this check wouldn't be highly required